### PR TITLE
Add GitHub Actions deployment workflow for deploy_dev

### PR DESCRIPTION
**Pull request by bbnickell**
@facing-n Adding worfklow for dev, had minor tweaks to the README during testing.

### CODE REVIEW

**Comment by bbnickell** [comment]
@facing-n once this gets approved I'll merge and proceed with rolling this out for dev.

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /home/ec2-user/nina-indexer
+pm2 stop nina-indexer && pm2 delete nina-indexer
+pm2 start yarn --interpreter bash --name nina-indexer -- start:indexer
```
Comment (facing-n) [inline run.sh:4]: we will also want to start api service here

**Review by facing-n — CHANGES_REQUESTED** [review]
just one note re: api service in run.sh

File: run.sh (hunk 1 above)
Comment (bbnickell) [reply run.sh]: @facing-n yep, added.

**Comment by bbnickell** [comment]
@facing-n updated.

**Review by facing-n — APPROVED** [review]

**Pull request merged**